examples/calibrate_test_signal.py

```diff
@@ -0,0 +1,122 @@
+"""
+Capture the signal acquired on the GDS. and converts to analytic 
+================================================================
+A single capture of the signal on the GDS is converted to analytic signal,
+frequency filtered and the absolute is taken to calculate the $V_{amp}$. The
+final plot that is generated plots the raw data, analytic data, filtered
+analytic data, and the absolute value and the caluclated $V_{amp}$ (in mV). 
+This script serves two purposes in calibrating test signal generated by the
+AFG: 
+
+1. Calculating the raw output of the AFG (no attenuator assemblies). For this
+step, simply set the calibrating_atten to False. Note that for the next step
+you need to set the input_Vamp to the value calculated in this step.
+
+2. Calculating the voltage ratio of the attenuator assemblies (placed between
+the AFG output and the GDS). Prior to running the script the user needs to
+have a rough estimate of the expected_Vamp when the attenuator assemblies are
+in place. The calculated voltage ratio and dB are printed on the final
+generated plot.
+"""
+from Instruments import GDS_scope
+from pyspecdata import figlist_var
+from pylab import text, gca
+
+calibrating_atten = True
+expected_Vamp = float(
+    input("What is the expected peak voltage (in units of V)?")
+)
+assert expected_Vamp < 500e-3, (
+    "That's way too high of a peak voltage! You either need an attenuator or"
+    " you didn't put the input in units of V"
+)
+if calibrating_atten:
+    print(
+        "Make sure the input_Vamp here is accurate so I can calculate the"
+        " right voltage ratio!"
+    )
+    input_Vamp = 500e-3  # this should be calculated prior to calibrating
+    #                      the attenuator assemblies
+    assert input_Vamp < 505e-3, (
+        "That's way too high of an input peak voltage! Make sure it's in units"
+        " of V and not mV!"
+    )
```

Comment:
gives a little leeway incase the actual is slightly larger than 1 V (GDS can handle 1.25 V)